opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -402,7 +391,7 @@ public void setGeoJsonSource(@NonNull FeatureCollection featureCollection, Featu
                     }
                 }
 
-                if (getInterventionLabel() == R.string.focus_investigation && revealMapHelper.getIndexCaseCircleLayer() == null) {
```

Comment:
This should be returned so that the line layer is drawn only once, then in subsequent times just changing the source

Reply:
This is also causing app to crash on changing an OA
```
com.mapbox.mapboxsdk.style.layers.CannotAddLayerException: Layer index-case-symbol-layer already exists
        at com.mapbox.mapboxsdk.maps.NativeMapView.nativeAddLayer(Native Method)
        at com.mapbox.mapboxsdk.maps.NativeMapView.addLayer(NativeMapView.java:769)
        at com.mapbox.mapboxsdk.maps.Style.addLayer(Style.java:174)
        at org.smartregister.reveal.util.RevealMapHelper.addIndexCaseLayers(RevealMapHelper.java:123)
```